sph/include/sph/particles_data.hpp

```diff
@@ -102,7 +99,6 @@ class ParticlesData
     std::vector<T>       c11, c12, c13, c22, c23, c33; // IAD components
     std::vector<T>       alpha;                        // AV coeficient
     std::vector<T>       xm;                           // Classical SPH density
```

Comment:
This is "volume element definition" instead of "Classical SPH density"

Reply:
ack